mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from "src/component/select/Select.module.scss";
+import {useState} from "react";
+
+interface SelectProps {
+  /**
+   * Label's text;
+   */
+  label: string;
+  /**
+   * Default select value (string)
+   */
+  value: string;
+  /**
+   * Select's name (string)
+   */
+  name: string;
+  /**
+   * Options list (Array<OptionType>)
+   */
+  options: OptionType[];
+}
```

Comment:
Don't repeat in () type of properties. Let's use DRY principle

Reply:
done

Reply:
I was mean that for example name: string (then  no need to repeat it in comment). Select's anme is enough. With options the same)

Reply:
* Options list 
   */
options: OptionType[];
is it enough?

Reply:
Maybe "Array of Options list" ?

Reply:
Guys,  i don't like to duplicate types in ts-docs, word list or even letter `s` in the end of word is enough

Reply:
got it 🙂